nservicebus/transports/scale-out.md

```diff
@@ -7,6 +7,10 @@ tags:
 - Routing
 ---
 
+## When to scale out
```

Comment:
Incorrect, scale-out can also be done for high availability or data redundancy reasons.